configs/dcgan/README.md

```diff
@@ -24,3 +24,4 @@
 | :---------: | :------------: | :----------------------: | :-----: | :-------------------------------------------------------------------------------------------------------------------: | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
 | DCGAN 64x64 | CelebA-Cropped |  8.93,10.53,50.32/23.26  | 0.2899  | [config](https://github.com/open-mmlab/mmgeneration/tree/master/configs/dcgan/dcgan_celeba-cropped_64_b128x1_300k.py) | [model](http://download.openmmlab.com/mmgen/dcgan/dcgan_celeba-cropped_64_b128x1_300kiter_20210408_161607-1f8a2277.pth) &#124; [log](http://download.openmmlab.com/mmgen/dcgan/dcgan_celeba-cropped_64_b128x1_300kiter_20210408_161607-1f8a2277.json) |
 | DCGAN 64x64 |  LSUN-Bedroom  | 42.79, 34.55, 98.46/58.6 | 0.2095  | [config](https://github.com/open-mmlab/mmgeneration/tree/master/configs/dcgan/dcgan_lsun-bedroom_64x64_b128x1_5e.py)  |         [model](http://download.openmmlab.com/mmgen/dcgan/dcgan_lsun-bedroom_64_b128x1_5e_20210408_161713-117c498b.pth) &#124; [log](http://download.openmmlab.com/mmgen/dcgan/dcgan_lsun-bedroom_64_b128x1_5e_20210408_161713-117c498b.json)         |
+| DCGAN 64x64 |  MNIST (64x64)| 21.16, 4.4, 8.41/11.32 | 0.1395  | [config](https://github.com/open-mmlab/mmgeneration/tree/master/configs/dcgan/dcgan_mnist-64_b128x1_Glr4e-4_Dlr1e-4_5k.py)  |         [model](https://openmmlab.oss-cn-hangzhou.aliyuncs.com/mmgen/dcgan/dcgan_mnist-64_b128x1_Glr4e-4_Dlr1e-4_5k_20210512_163926-207a1eaf.pth?versionId=CAEQKxiBgMDTnYH_yhciIGY2MjAwMzBjZjZiZjQzODJhZWRhOTFiNjU4MzdhMWY2) &#124; [log](https://openmmlab.oss-cn-hangzhou.aliyuncs.com/mmgen/dcgan/dcgan_mnist-64_b128x1_Glr4e-4_Dlr1e-4_5k_20210512_163926-207a1eaf.json?versionId=CAEQKxiBgMDOnYH_yhciIDg5NDUwMWFhYWZhZTQ5ZWQ4ZDA1MTIzODE5OTVlNGNk)         |
```

Comment:
You should change the address of `https://openmmlab.oss-cn-hangzhou.aliyuncs.com/` to `http://download.openmmlab.com/`.